packages/nimble/R/cppDefs_cppProject.R

```diff
@@ -143,7 +143,7 @@ cppProjectClass <- setRefClass('cppProjectClass',
                                        selfCPP <- if(is.character(con)) paste0('"', con, '.cpp"') else '"[FILENAME].cpp"'
                                        CPPincludes <- CPPincludes[ CPPincludes != selfCPP ]
 
-                                       ## Eigen must be included before any R header files because they both define "length"
+                                       ## Eigen must be included before any R header files because they both define "length" (this may be moot, 7/17)
```

Comment:
As of #325, this should no longer be an issue. Consider changing this comment to
```r
## TODO Simplify Eigen include logic now that Nimble defines `R_NO_REMAP`.
```